modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/SourceProcessorTestParameters.java

```diff
@@ -40,6 +55,16 @@ public SourceProcessorTestParameters addExpectedMessage(
 		return this;
 	}
 
+	public SourceProcessorTestParameters addExpectedMessage(String[] messages) {
+		Map<String, Integer> messagesMap = new LinkedHashMap<>();
+
+		for (String message : messages) {
+			messagesMap.put(message, -1);
+		}
+
+		return addExpectedMessage(messagesMap);
+	}
+
```

Comment:
@nicolas-sss, why don't we do only the following:
```
public SourceProcessorTestParameters addExpectedMessages(String[] messages) {
	for (String message : messages) {
		addExpectedMessage(message, -1);
	}

	return this;
}
```

Reply:
Makes sense hahha